searchlib/src/vespa/searchlib/attribute/stringbase.cpp

```diff
@@ -86,9 +86,10 @@ StringAttribute::getFloat(DocId doc) const {
 }
 
 vespalib::ConstArrayRef<char>
-StringAttribute::get_raw(DocId) const
+StringAttribute::get_raw(DocId doc) const
```

Comment:
Some of the StringAttribute unit tests should be extended to use this API as well.

Reply:
Done.